src/qt/modaloverlay.h

```diff
@@ -9,7 +9,7 @@
 #include <QWidget>
 
 //! The required delta of headers to the estimated number of available headers until we show the IBD progress
-static const int REQ_HEADER_HEIGHT_DELTA_SYNC = 24;
+static constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
```

Comment:
It is not like the number represents a *required* delta, it is basically a more or less arbitrarily chosen delta.